worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitBlockCommandSender.java

```diff
@@ -146,16 +147,34 @@ public BlockCommandSender getSender() {
     @Override
     public SessionKey getSessionKey() {
         return new SessionKey() {
+
+            private volatile boolean active = true;
+
+            private void updateActive() {
```

Comment:
this at least needs to check if the chunk is loaded. unloaded block = inactive block.